vocabularies/voc4cat/0007021.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007021>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007021"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007003> ;
+    skos:definition "The bulk volume taken up by the catalyst in a reactor."@en ;
+    skos:historyNote "0000-0002-4041-0165 Julia Schumann created the resource"@en ;
+    skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
+    skos:prefLabel "apparent catalyst volume"@en ;
+.
```

Comment:
Is there some chem.eng. literature that defines/uses this term?

I also heard "catalyst bed volume" for this. How to deal with bed dilution? Does the apparent catalyst volume increase by diluting the catalyst with inert material?

Reply:
We have something called "Schüttvolumen". I thought this was the closest translation, but "catalyst bed volume" also sounds good. Yes, I think the volume increases with dilution by an inert material. I will double check and specify the definition.

Reply:
I move the term 'apparent catalyst volume' to the alternate label and use 'catalyst bed volume' as preferred label now.